geowebcache/pom.xml

```diff
@@ -561,7 +571,6 @@
       </plugin>
 
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Interesting, what's up here? I guess this groupid is a default, when talking about maven plugins?

Reply:
yes it is